packages/plugin-ebarimt-api/src/utils.ts

```
@@ -384,7 +384,16 @@ const calcProductsTaxRule = async (subdomain: string, models: IModels, config, p
   }
 }
 
-export const getPostData = async (subdomain, models: IModels, config, deal) => {
+export const mathRound = (value, p = 2) => {
+  if (!value || isNaN(Number(value))) {
+    return 0;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)